File: 백태균/week03/12950_행렬의_덧셈.java
class Solution {
    public int[][] solution(int[][] arr1, int[][] arr2) {
        int[][] answer = new int[arr1.length][arr1[0].length];  // 결과 행렬

        // 2차원 배열이니깐 이중 for문 순회
        for (int i=0; i<arr1.length; i++) {
            for (int j=0; j<arr1[0].length; j++) {
                answer[i][j] = arr1[i][j] + arr2[i][j];  // arr1와 arr2 같은 행, 같은 열의 값을 더한 값을 결과 행렬에 더해서 추가
            }
        }

        return answer;
    }
}

File: 김진효/week03/12950_행렬의_덧셈.py
 # [성능 요약] 메모리: 22 MB 시간: 32.21 ms 

def solution(arr1, arr2):
    answer = []
    for i in range(len(arr1)):
        temp = []
        for j in range(len(arr1[i])):
            temp.append(arr1[i][j]+arr2[i][j])
        answer.append(temp)
    return answer